### PR TITLE
Add the blog for announcing Vitess 16

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

This PR adds the blog for announcing Vitess 16.

### Related Issues

- https://github.com/vitessio/vitess/issues/12293

### CODE REVIEW

File: content/en/blog/2023-02-28-announcing-vitess-16.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+author: 'Vitess Maintainer Team'
+date: 2023-02-28
+slug: '2023-02-28-announcing-vitess-16'
+tags: ['release','Vitess','MySQL','kubernetes','operator','cloud','GKE','sharding']
```
Comment (mattlord) [inline content/en/blog/2023-02-28-announcing-vitess-16.md]: Not a problem, but curious how we came up with this list. GKE e.g. We could add Orchestration, Failover, HA, etc for the vtorc and Backup for PITR

File: content/en/blog/2023-02-28-announcing-vitess-16.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+We have been making steady progress on adding query support for more MySQL constructs. In this release we have added support for Views in Vitess. 
+It is now possible to create views that access data across shards and they will work as intended in Vitess. Note that this is considered an experimental feature. It will move to GA in a future release.
+
+## [Other improvements](https://github.com/vitessio/vitess/releases/tag/v16.0.0)
+
+Support for native incremental backups and point in time recoveries has been added. It is now possible to take an incremental backup, starting with the last known (full or incremental) backup, 
+and up to either a specified (GTID) position, or the current ("auto") position. Using these incremental backups it is then possible to restore a backup up to a given point in time (GTID position) without 
+relying on a binlog server. Note that this is only supported for the file-based builtin backup method, not for xtrabackup.
+A [new `VEXPLAIN` command](https://vitess.io/docs/16.0/user-guides/sql/vexplain/) has been introduced to help the users gain more insight into query planning in Vitess.
```
Comment (mattlord) [inline content/en/blog/2023-02-28-announcing-vitess-16.md:41]: I can’t tell how these items will be separated? Should they be bullet points?

**Review by mattlord — APPROVED** [review]
LGTM! Just a couple of minor notes.

File: content/en/blog/2023-02-28-announcing-vitess-16.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/blog/2023-02-28-announcing-vitess-16.md]: Oh yes, I just copied whatever we had for `Announcing v15`

File: content/en/blog/2023-02-28-announcing-vitess-16.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/blog/2023-02-28-announcing-vitess-16.md:41]: They should be separate paragraphs. Let me fix it.

File: content/en/blog/2023-02-28-announcing-vitess-16.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/blog/2023-02-28-announcing-vitess-16.md]: Added them all.

File: content/en/blog/2023-02-28-announcing-vitess-16.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/blog/2023-02-28-announcing-vitess-16.md:41]: Fixed

**Review by frouioui — APPROVED** [review]

**Review by shlomi-noach — APPROVED** [review]

File: content/en/blog/2023-02-28-announcing-vitess-16.md (hunk 1 above)
Comment (deepthi) [reply content/en/blog/2023-02-28-announcing-vitess-16.md]: someone sometime came up with a list and we've just been copying it for every new release 😄

File: content/en/blog/2023-02-28-announcing-vitess-16.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+We are marking [VDiff v2](https://vitess.io/docs/16.0/reference/vreplication/vdiff/) as Generally Available or production-ready in v16. 
+We now recommend that you use v2 rather than v1 going forward. Version 1 will be deprecated and eventually removed in future releases.
+
+This new version of VDiff should offer a much improved overall user experience, especially when migrating very large tables. 
+You can read more about VDiff v2 in the [Introducing VDiff V2 blog post](https://vitess.io/blog/2022-11-22-vdiff-v2/).
+
+## VTOrc is now mandatory
+
+[VTOrc](https://vitess.io/blog/2022-09-21-vtorc-vitess-native-orchestrator/) is a required component of Vitess starting from this release. 
+It is required to run at least one instance of VTOrc in order for Vitess to automatically manage the backing MySQL clusters.
```
Comment (deepthi) [inline content/en/blog/2023-02-28-announcing-vitess-16.md]: We are repeating `required` in these two sentences. Maybe change the second one to `necessary`?

**Review by deepthi — APPROVED** [review]
Looks really good. Just one small suggestion.

File: content/en/blog/2023-02-28-announcing-vitess-16.md (hunk 3 above)
Comment (GuptaManan100) [reply content/en/blog/2023-02-28-announcing-vitess-16.md]: Fixed

**Pull request merged**